UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -535,6 +535,50 @@ public void TestGenericCast()
             Assert.Equal(abiView.ThisPtr, abiView.As<WinRT.IInspectable>().As<ABI.System.Collections.Generic.IReadOnlyList<int>.Vftbl>().ThisPtr);
         }
 
+        [ComImport]
+        [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+        [Guid("96369F54-8EB6-48F0-ABCE-C1B211E627C3")]
+        internal unsafe interface IStringableInterop
+        {
+            // Note: ComInterfaceType.InterfaceIsIInspectable no longer supported
+            void GetIids(out int iidCount, out IntPtr iids);
+            void GetRuntimeClassName(out IntPtr className);
+            void GetTrustLevel(out TrustLevel trustLevel);
+
+            void ToString(out IntPtr hstr);
+        }
+
+        [ComImport]
+        [Guid("39E050C3-4E74-441A-8DC0-B81104DF949C")]
+        [InterfaceType(ComInterfaceType.InterfaceIsIInspectable)]

```

Comment:
I see above comment says ComInterfaceType.InterfaceIsIInspectable is not supported, but is in use here.  Am I misunderstanding something?

Reply:
I'll clarify the comment